msal/src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -114,6 +114,7 @@ protected override string GetUIBehaviorPromptValue()
                         AuthenticationRequestParameters.RequestContext)
                         .ConfigureAwait(false);
                 uiEvent.UserCancelled = _authorizationResult.Status == AuthorizationStatus.UserCancel;
+                uiEvent.AccessDenied = _authorizationResult.Status == AuthorizationStatus.ProtocolError;
```

Comment:
when I was testing embedded webview, w/corp account and getting the protocolError, it was getting registered as a UserCancelled, even though it was AccessDenied in the telemetry. I added in this line to register the protocolError.

Reply:
Why call it access denied if it's a protocol error?

---
In reply to: [200799703](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1114#discussion_r200799703) [](ancestors = 200799703)

Reply:
The error that is returned is `access_denied`, and we equate it with an `AuthorizationStatus.ProtocolError` [here](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/blob/dev/core/src/UI/AuthorizationResult.cs#L114), but seems like `access_denied` is more specific, which is why I included that for the telemetry, but can change it to protocol error.